recipes/tensorflow-addons/build.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -ex
+
+export PATH="$PWD:$PATH"
+export CC=$(basename $CC)
+export CXX=$(basename $CXX)
+export LIBDIR=$PREFIX/lib
+export INCLUDEDIR=$PREFIX/include
+
+export TF_NEED_CUDA=0
+
+if [[ ${cuda_compiler_version} != "None" ]]; then
+    export TF_NEED_CUDA=1
+    export TF_CUDA_PATHS="${PREFIX},${CUDA_HOME}"
+    export TF_CUDA_VERSION="${cuda_compiler_version}"
+    export TF_CUDNN_VERSION="${cudnn}"
+    export GCC_HOST_COMPILER_PATH="${GCC}"
+    export CUDA_TOOLKIT_PATH="${CUDA_HOME}"
+    export CUDNN_INSTALL_PATH="${PREFIX}"
+
+    # addons uses a different nomenclature for compute capabilities than tf proper, see
+    # https://github.com/tensorflow/addons/blob/v0.14.0/build_deps/toolchains/gpu/cuda_configure.bzl#L18-L19
+    if [[ ${cuda_compiler_version} == 10.* ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=3.5,5.0,6.0,6.2,7.0,7.2,7.5
+    elif [[ ${cuda_compiler_version} == 11.0* ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=3.5,5.0,6.0,6.2,7.0,7.2,7.5,8.0
+    elif [[ ${cuda_compiler_version} == 11.1 ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=3.5,5.0,6.0,6.2,7.0,7.2,7.5,8.0,8.6
+    elif [[ ${cuda_compiler_version} == 11.2 ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=3.5,5.0,6.0,6.2,7.0,7.2,7.5,8.0,8.6
+    else
+        echo "unsupported cuda version."
+        exit 1
+    fi
+fi
+
+# This script links project with TensorFlow dependency; creates .bazelrc
+python ./configure.py
+# ironically, configure.py cannot be configured, so sed it is...
+sed -i'' -e "s:TF_HEADER_DIR=.*:TF_HEADER_DIR=\"$PREFIX/include\":" \
+    -e "s:TF_SHARED_LIBRARY_DIR=.*:TF_SHARED_LIBRARY_DIR=\"$PREFIX/lib\":" \
+    .bazelrc
+# show result (& sleep to allow log to catch up)
+cat .bazelrc
+sleep 2
+
+bazel build build_pip_pkg
```

Comment:
Can you add the `bazel-toolchain` here? CUDA changes have been merged and applied for `jaxlib`, so this should be good to go here, too.

Reply:
> Can you add the `bazel-toolchain` here?

What does "add" mean? I changed `bazel` to `bazel-toolchain`, but that didn't work.